src/org/rascalmpl/library/util/PathConfig.java

```diff
@@ -417,14 +418,20 @@ private static void buildRascalConfig(ISourceLocation workspaceRascal, RascalCon
                 typepal = resolveProjectOnClasspath("typepal");
                 typepal = MavenRepositoryURIResolver.mavenize(typepal);
                 typepal = JarURIResolver.jarify(typepal);
-            } catch (FileNotFoundException e) {
-                messages.append(Messages.error("Could not find typepal in local project, rascal compiler will not work", workspaceRascal));
+            } 
+            catch (FileNotFoundException e) {
+                // last try: the dependencies. This should typically succeed while we have a pom dependency on typepal in the rascal project.
+                typepal = getPomXmlTypePalDependency(workspaceRascal, messages);
```

Comment:
this is not needed, it's already in the argument `List<Artifact> mavenClassPath`